PIL/ImageFont.py

```diff
@@ -169,6 +169,15 @@ def font_variant(self, font=None, size=None, index=None, encoding=None):
                             encoding=self.encoding if encoding is None else
                             encoding)
 
+    def getBB( self, text ):
```

Comment:
What's a BB? Could this be named more descriptively? A docstring (`"""TODO"""`) would be useful.